src/Plivo/Resources/Message/MessageInterface.php

```diff
@@ -49,6 +49,12 @@ public function get($messageUuid)
             $this->uri . $messageUuid .'/',
             []
         );
+        // return the object for chain method 
```

Comment:
This looks like an API change i.e modification and not mere addition and hence you might have to bump up the major version.

Reply:
> This looks like an API change i.e modification and not mere addition and hence you might have to bump up the major version.

this chain method was there very long back, I just did a fix on invoke, So upgraded the Minor version. When we return a object, it was printing whole history including client, properties, other params, etc etc. so added __debugInfo, which will return what we required on output.